packages/debug/src/browser/style/index.css

```diff
@@ -156,12 +156,12 @@
 }
 
 .debug-tab-icon {
-    -webkit-mask: url('debug-dark.svg') 50%;
+    -webkit-mask: url('debug-dark.svg') 50% 50% / var(--theia-private-sidebar-icon-size) no-repeat;
```

Comment:
Could you elaborate on this change?
Should we align mask as well?
What if the widget is moved to the main or bottom area? Does icon still look good?

Reply:
I've just copied that style from VSCode. Previously it set a fixed size
```
width: var(--theia-private-sidebar-icon-size);
height: var(--theia-private-sidebar-icon-size);
```
and it was just about 24px, while toolbar size is 50px, so I replaced it with bigger size, and removed padding, which I suppose used to keep icon in the center.
```
  width: var(--theia-private-sidebar-tab-width);
  height: var(--theia-private-sidebar-tab-width);
```
So, now it supposed to be placed in the center with the `-webkit-mask`, and have a size as previously, but it now affects only for width while height as auto.
I'll extend the to `mask` as well.  Just checking how it looks in Firefox.

> What if the widget is moved to the main or bottom area? Does icon still look good?
I don't know how to do it, if you would give me info, how to do it, I will check

Reply:
Main 
![image](https://user-images.githubusercontent.com/1212251/91234996-32d40b00-e73d-11ea-85bf-6c61c1f79df8.png)
Bottom
![image](https://user-images.githubusercontent.com/1212251/91235026-47180800-e73d-11ea-907b-aede765dae23.png)
RIght
![image](https://user-images.githubusercontent.com/1212251/91235068-60b94f80-e73d-11ea-9600-3156e56d37b8.png)